modules/private_data_fusion_network/main.tf

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+provider "google" {
+  version = "~> 3.0"
+}
+
+data "google_client_config" "current" {}
+
+resource "google_project_service" "project" {
+  project = var.project_id
+  service = "datafusion.googleapis.com"
+
+  disable_dependent_services = true
+}
+
+resource "google_compute_network_peering" "peering1" {
+  name         = "data-fusion-peering"
+  network      = var.vpc_network_self_link
+  peer_network = "projects/${var.tenant_project}/global/networks/${var.region}-${var.instance}"
+  import_custom_routes = "true"
+  export_custom_routes = "true"
+}
+
+resource "google_compute_subnetwork" "dataproc" {
```

Comment:
Please use the network module - https://github.com/terraform-google-modules/terraform-google-network

Reply:
I might be doing something wrong but I have issues using the submodules for firewall and peering. Should I not be using them directly like this?

```

Error: Module not found

Module "datafusion_firewall" (from main.tf:62) cannot be found in the module
registry at registry.terraform.io.


Error: Module not found

Module "peering" (from main.tf:44) cannot be found in the module registry at
registry.terraform.io.
```

Reply:
Looks like the README is incorrect. You should use `terraform-google-modules/network/google//modules/network-peering`.

Reply:
thanks!